### PR TITLE
モデル選択でPSPnetを、ロス選択でjaccard lossを選択可能に

### PR DESCRIPTION
**Pull request by muramasa2**
segmentation-config.iniにおいて
loss = jaccard_loss, cce_jaccard_loss, categorical_focal_jaccard_loss が選択可能に
model_name = pspnet が選択可能に(pspnetは画像入力サイズに制限があるため注意(エラー文参照))

### CODE REVIEW

File: apps/domain/tasks/build_model_task.py [hunk 1]
```diff
@@ -120,12 +131,23 @@ def _do_compile_model_task(
...
                     loss = losses.dice_loss
                 elif loss_func == "categorical_focal_dice_loss":
                     loss = losses.categorical_focal_dice_loss
+
+                elif loss_func == "cce_jaccard_loss":
+                    loss = losses.cce_jaccard_loss
+                elif loss_func == "jaccard_loss":
+                    loss = losses.jaccard_loss
+                elif loss_func == "categorical_focal_jaccard_loss":
+                    loss = losses.categorical_focal_jaccard_loss
```
Comment (aiorhiroki) [inline apps/domain/tasks/build_model_task.py]: `elif`だらけだけど、これは何か良い書き方ないのかな〜

Comment (muramasa2) [reply apps/domain/tasks/build_model_task.py]: リスト格納してとかくらいしか思いつかないですけど、それでも大して綺麗にはならないのでなんかいい方法あればいいですがパッと思いつかないですorz（コーディング力がほしい）

File: apps/domain/tasks/build_model_task.py [hunk 2]
```diff
@@ -120,18 +139,14 @@ def _do_compile_model_task(
...
-                else:
-                    raise NotImplementedError
+                print('------------------')
+                print('Loss:', loss_func)
+                print('------------------')
                 model.compile(
                     optimizer=optimizer,
-                    loss=loss,
-                    metrics=[metrics.iou_score, losses.categorical_crossentropy],
+                    loss=globals()[loss_func],
```
Comment (aiorhiroki) [inline apps/domain/tasks/build_model_task.py]: こんな感じでできた

Comment (muramasa2) [reply apps/domain/tasks/build_model_task.py]: これスマートですね！
流石っす

Comment (aiorhiroki) [reply apps/domain/tasks/build_model_task.py]: kerasのソースコード参考にした。
lossとかoptimizerとか、同じ仕組みだし。
